uint32 timestamp
uint8 n_estimates

# Estimate of doas in degrees
float32[] doa_estimates_deg
